Userland/Libraries/LibWeb/CSS/StyleValues/FilterValueListStyleValue.cpp

```diff
@@ -18,7 +18,7 @@ float Filter::Blur::resolved_radius(Layout::Node const& node) const
     // Default value when omitted is 0px.
     auto sigma = 0;
     if (radius.has_value())
-        sigma = radius->to_px(node).value();
+        sigma = radius->to_px(node).to_double();
```

Comment:
```suggestion
        sigma = radius->to_px(node).to_int();
```
(Note that sigma will be an int from the `auto sigma = 0`